src/amo/components/LanguagePicker.scss

```diff
@@ -7,6 +7,8 @@ $selector-padding: 10px;
 }
 
 .LanguagePicker-selector {
+  -moz-appearance: none;
```

Comment:
Ah, right on. Any reason we don't have that on in development? It's definitely handy and doesn't really add much bloat...